tests/test_workflows.py

```diff
@@ -0,0 +1,79 @@
+import csv
+
+from dsaps import workflows
+
+
+def test_create_file_dict_and_id_list(runner, sample_files_dir):
+    """Test create_file_dict_and_id_list function."""
+    file_path = sample_files_dir
+    file_dict, file_ids = workflows.create_file_dict_and_list(sample_files_dir,
+                                                              'pdf')
+    assert file_dict['test_02'] == f'{file_path}/test_02.pdf'
+    assert file_dict['test_01'] == f'{file_path}/test_01.pdf'
+    assert file_dict['best_01'] == f'{file_path}/best_01.pdf'
+    for id in ['test_02', 'test_01', 'best_01']:
+        assert id in file_ids
+
+
+def test_create_metadata_id_list(runner, sample_files_dir):
+    """Test create_metadata_id_list function."""
+    metadata_path = f'{sample_files_dir}/metadata.csv'
+    metadata_ids = workflows.create_metadata_id_list(metadata_path)
+    assert 'test' in metadata_ids
```

Comment:
Wouldn't hurt to test the entire contents of `metadata_ids` here, especially since it's only two items long.